frontend/src/container/SearchResult.jsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import {
+  PageWrapper,
+  PageTitle,
+  PageContent,
+  ListItemWrapper,
+  displayDistanceDate,
+  IconButton,
+  BREADCRUMBS_TYPE
+} from 'tracim_frontend_lib'
+import {
+  PAGE,
+  ALL_CONTENT_TYPES
+} from '../helper.js'
+import ContentItemSearch from '../component/ContentItemSearch.jsx'
+import ContentItemHeader from '../component/Workspace/ContentItemHeader.jsx'
+import {
+  newFlashMessage,
+  setCurrentNumberPage,
+  setCurrentNumberSearchResults,
+  appendSearchResultsList,
+  setBreadcrumbs
+} from '../action-creator.sync.js'
+import { getSearchedKeywords } from '../action-creator.async.js'
+
+class searchResult extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      showArchived: 0,
+      contentTypes: ALL_CONTENT_TYPES,
+      showDeleted: 0,
+      showActive: 1
+    }
+  }
+
+  customEventReducer = ({ detail: { type, data } }) => {
+    switch (type) {
+      case 'allApp_changeLang': this.buildBreadcrumbs(); break
+    }
+  }
+
+  componentDidMount () {
+    this.buildBreadcrumbs()
+  }
+
+  findPath = (parentsList) => {
+    let parentPath = ''
+    if (parentsList.length > 0) {
+      parentPath = parentsList.reduce((acc, currentParent) => `${currentParent.label}/${acc}`, '')
+    }
+    return parentPath
+  }
+
+  putContentName = (content) => {
+    // FIXME - GB - 2019-06-04 - we need to have a better way to check if it is a file than using contentType[1]
+    // https://github.com/tracim/tracim/issues/1840
+    const { props } = this
+
+    if (props.contentType.length > 1) {
+      return content.content_type === props.contentType[1].slug ? content.filename : content.label
+    } else {
+      props.dispatch(newFlashMessage(props.t('An error has happened'), 'error'))
```

Comment:
if your function should return a string, it must return a string in every cases.
So add a `return ''` in the `else`